consensus/misc/eip1559/eip1559.go

```diff
@@ -141,22 +187,25 @@ func CalcBaseFee(config *params.ChainConfig, parent *types.Header, time uint64)
 	}
 	elasticity := config.ElasticityMultiplier()
 	denominator := config.BaseFeeChangeDenominator(time)
-	if config.IsHolocene(parent.Time) {
+	var minBaseFee uint64
+	if config.IsConfigurableMinBaseFee(parent.Time) {
```

Comment:
It might be nice to encapsulate this into a single validation function which takes the parent block time and does the fork/feature switching. That would allow us to put all the OPStack-specific code in a separate file and keep the diff to upstream geth super clean. Establishing this pattern now prevents a messier diff as we add do more forks.